source/install-desktops/modify-cluster-configuration.rst

```diff
@@ -0,0 +1,53 @@
+.. _install-desktops-modify-cluster-configuration:
+
+Modify Cluster Configuration
+============================
+
+If you haven't already done so, you will need to first have a corresponding
+cluster configuration file for the cluster you intend to launch desktops on. It
+is recommended you follow the directions on :ref:`add-cluster-config`.
```

Comment:
I believe this is a requirement though for the core apps to work. So it may be too much detail for this, but your call.